DEVELOPMENT.md

```diff
@@ -0,0 +1,76 @@
+# Development
+
+## Prerequisites
+
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [Go](https://golang.org/doc/install)
+* [Docker](https://www.docker.com/)
+* Make
+    * macOS: `xcode-select --install`
+    * Windows: `choco install make`
+
+## Tasks
+
+### Building
+
+To build pack:
+```
+make build
+```
+
+This will output the binary to the directory `out/`.
+
+Options:
+
+| ENV_VAR      | Description                                                            | Default |
+|--------------|------------------------------------------------------------------------|---------|
+| GOCMD        | Change the `go` executable. For example, [richgo][rgo] for testing.    | go      |
+| PACK_BIN     | Change the name or location of the binary relative to `out/`.          | pack    |
+| PACK_VERSION | Tell `pack` what version to consider itself                            | `dev`   |
+
+[rgo]: https://github.com/kyoh86/richgo
+
+_NOTE: This project uses [go modules](https://github.com/golang/go/wiki/Modules) for dependency management._
+
+### Testing
+
+To run unit and integration tests:
+```shell
+make unit
+```
+
+To run acceptance tests:
+```shell
+make acceptance
```

Comment:
Just a comment, I notice we have no mention of `make acceptance-all` (compat). This is probably for the best, until we finish our current work to download all the assets automatically. Maybe we can make documentation of that process a part of the current work.

Reply:
Definitely "for the best". I agree with your assessment.